docs/build/autogenerate.rst

```diff
@@ -747,20 +747,29 @@ Above, we configure ``hooks`` to be a single post write hook labeled
 configuration for the ``"black"`` post write hook, which includes:
 
 * ``type`` - this is the type of hook we are running.  Alembic includes
-  a hook runner called ``"console_scripts"``, which is specifically a
+  two hook runners: ``"console_scripts"``, which is specifically a
   Python function that uses ``subprocess.run()`` to invoke a separate
-  Python script against the revision file.  For a custom-written hook
-  function, this configuration variable would refer to the name under
+  Python script against the revision file; and ``"exec"``, which uses
+  ``subprocess.run()`` to execute an arbitrary binary.  For a custom-written
+  hook function, this configuration variable would refer to the name under
   which the custom hook was registered; see the next section for an example.
```

Comment:
**mike bayer** (zzzeek) wrote:

needs ".. versionadded <target version>: added new exec runner"

@caselit do we want to use major release version for this as it's totally new feature?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4788

Reply:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4788